packages/flutter/lib/src/widgets/asset_vendor.dart

```diff
@@ -0,0 +1,200 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:mojo/core.dart' as core;
+
+import 'basic.dart';
+import 'framework.dart';
+
+// Base class for asset resolvers.
+abstract class _AssetResolver {
+  // Return a resolved asset key for the asset named [name].
+  Future<String> resolve(String name);
+}
```

Comment:
Oh, I didn't notice that.  Yes, the doc generator knows how to skip private classes and methods.  If its private, I'm not sure it matters which kind of comment we use.  Sorry for the noise.